src/components/cards/horizontalCard/HorizontalCard.jsx

```diff
@@ -2,7 +2,33 @@ import React from "react";
 import "./horizontalCard.css";
 
 function HorizontalCard() {
```

Comment:
Suggestion: You can make one Card Component and pass in prop = { "horizntal", "vertical" , large, small ... }
problem: code duplication, and difficult to maintain in the feature

Reply:
Yaah it can be done, but there are many style changes thats why made a different one .